work/worker.go

```diff
@@ -533,11 +533,18 @@ func (self *worker) commitNewWork() {
 		// wait for a while and get a new timestamp
 		if tstart.Before(ideal) {
 			wait := ideal.Sub(tstart)
-			logger.Info("Mining too far in the future", "wait", common.PrettyDuration(wait))
+			logger.Debug("Mining too far in the future", "wait", common.PrettyDuration(wait))
 			time.Sleep(wait)
 
 			tstart = time.Now()    // refresh for metrics
 			tstamp = tstart.Unix() // refresh for block timestamp
+		} else {
```

Comment:
Doesn't it `else if tstart.After(deal)`? Ideally, if this log is for a time that is far from the ideal time, push log it as suggested below, if not (ideal time), do not log. Correct?

Reply:
You're right. Although `tstart==ideal` is extremely difficult because they have to match the nanoseconds. However, I'll add the condition to be more explicit.

Reply:
fixed in 4a0d746d